src/governance-classes.cpp

```diff
@@ -512,9 +509,31 @@ CSuperblock(uint256& nHash)
     DBG( cout << "CSuperblock Constructor End" << endl; );
 }
 
-void 
-CSuperblock::
-ParsePaymentSchedule(std::string& strPaymentAddresses, std::string& strPaymentAmounts)
+bool CSuperblock::IsValidBlockHeight(int nBlockHeight)
```

Comment:
The idea was that some abstract superblock should be aware of some general restrictions like allowed blockheight and payments limit and should not ask some outside class for this kind of info. Manager imo should only keep a list, create, remove etc., it should ask is superblock if it's valid (in general / for some height) and it's superblocks job to answer correctly.